src/Microsoft.DotNet.Interactive.Formatting/HtmlFormatter.cs

```diff
@@ -226,22 +226,7 @@ void Initialize()
             var formatter = GetDefaultFormatterForAnyObject(type);
```

Comment:
Looks like most lambdas in this collection don't guard against the `value` being `null` (although the removed last resort one below was checking for null). Is this intentional or does some other piece of code else already filter out nulls before these lambdas are invoked?

Reply:
Matching is done based on the runtime type and the formatter provides a universal early out for `null`.